ramp-database/rampdb/model/submission.py

```diff
@@ -114,8 +206,14 @@ def __init__(self, name, event_team):
         # url which is maybe not a good idea.
         self.hash_ = '{}'.format(sha_hasher.hexdigest())
         self.submission_timestamp = datetime.datetime.utcnow()
-        event_score_types = EventScoreType.query.filter_by(
-            event=event_team.event)
+        if session is None:
```

Comment:
a little bit above, `self.session` is already defined by inspecting the `event_team` object